src/assets/styles/includes.scss

```diff
@@ -30,4 +30,8 @@ calcite-tab[is-active] {
 
 a {
   color: $ui-blue;
-}
\ No newline at end of file
+}
+
+.hydrated--invisible {
+  visibility: hidden;
```

Comment:
Is this needed for popover positioning?

Reply:
correct. The popover needs to be hidden initially and then shown when open. However, Stencil adds a `visibility:true` to `hydrated` for the flash of unstyled content thing. I wish there was a per component way to disable this. I'll open a support ticket for them. Does that sound good? @jcfranco

Reply:
👍

Reply:
Created ticket for Stencil.